wallet/src/Ergvein/Wallet/Page/Settings.hs

```diff
@@ -78,9 +89,38 @@ settingsPage = do
             GoMnemonicExport mnemonic -> mnemonicExportPage mnemonic
             GoDns                     -> dnsPage
             GoNodes                   -> btcNodesPage
+            GoPassword                -> passwordChangePage
         , retractablePrev = Just $ pure settingsPage
         }
 
+passwordChangePage :: MonadFront t m => m ()
+passwordChangePage = do
+  passE <- changePasswordWidget
+  authInfoD <- getAuthInfo
+  eaibE <- withWallet $ ffor passE $ \(pass, b) prv -> do
+    ai <- sampleDyn authInfoD
+    encryptPrvStorageResult <- encryptPrvStorage prv pass
+    case encryptPrvStorageResult of
+      Left err -> pure $ Left $ CreateStorageAlert err
+      Right prve -> case passwordToECIESPrvKey pass of
+        Left _ -> pure $ Left GenerateECIESKeyAlert
+        Right k -> pure $ Right $ (,b) $ ai
+          & authInfo'storage . storage'encryptedPrvStorage .~ prve
+          & authInfo'eciesPubKey .~ toPublic k
+          & authInfo'isPlain .~ (pass == "")
+  aibE <- handleDangerMsg eaibE
+#ifdef ANDROID
```

Comment:
Lets do `when isAdnroid`, it is more maintainable as compilation and warnings are produced for build on any platform.